### ISSUE TITLE
Incorrect documentation

### ISSUE THREAD

**Issue by IsaiahJTurner** [mixpanel/mixpanel-swift#381]
> Call Reset on Logout
> 
> Reset generates a new random distinct_id and clears super properties. Call reset to clear data attributed to a user when that user logs out. This allows you to handle multiple users on a single device. For more information about maintaining user identity, see the Identity Management: Best Practices article.
> 
> Generate a non-IFA distinct_id by setting the MIXPANEL_RANDOM_DISTINCT_ID pre-processor flag and calling reset on logout. Calling reset will only clear the existing distinct_id from the cookie (browser) or local storage (mobile). You can set the MIXPANEL_RANDOM_DISTINCT_ID=1 pre-processor flag in your Active Compilation Conditions on the Mixpanel framework target.
> 
> After you call reset, Mixpanel generates a new distinct_id. If the MIXPANEL_RANDOM_DISTINCT_ID=1 preprocessor flag is set, the SDK will generate a non-IFA random persistent UUI, to use as the distinct_id. The SDK will ensure that all distinct_ids will not be the IFA, and will use the UUID. Setting the MIXPANEL_RANDOM_DISTINCT_ID=1 preprocessor flag and calling reset ensures that multiple users on the same device are not assigned the same alias.

The statement "Reset generates a new random distinct_id and clears super properties." is misleading. A developer wishing to handle sign out will overlook

**Comment by IsaiahJTurner** [comment]
`MIXPANEL_RANDOM_DISTINCT_ID` should be available on `init`, not as a preprocessor flag, as this is a non-standard way to write code and would not be considered by a developer reading the Full API documentation (not the summary HTTP version).

With the advent of Identity Merge, the default behavior of Mixpanel should be **not** to use the IDFA as this will cause profiles to be merged together incorrectly. If this change will not be made, the docs should SCREAM AT YOU that if you want to  handle multiple users on the same device, you will need to set a compiler flag. Use one of those big red warning boxes or something.

**Comment by IsaiahJTurner** [comment]
Put this in your Podfile if you are experiencing user profiles merged together after sign out:

```
  post_install do |installer|
      installer.pods_project.targets.each do |target|
          if target.name == 'Mixpanel-swift'
              target.build_configurations.each do |config|
                config.build_settings['OTHER_SWIFT_FLAGS'] = '$(inherited) -D COCOAPODS -D DECIDE -D MIXPANEL_RANDOM_DISTINCT_ID'
              end
          end
      end
  end
```

**Comment by IvanovDeveloper** [comment]
Thank you a lot. It works for me.

**Comment by IvanovDeveloper** [comment]
> Put this in your Podfile if you are experiencing user profiles merged together after sign out:
> 
> ```
>   post_install do |installer|
>       installer.pods_project.targets.each do |target|
>           if target.name == 'Mixpanel-swift'
>               target.build_configurations.each do |config|
>                 config.build_settings['OTHER_SWIFT_FLAGS'] = '$(inherited) -D COCOAPODS -D DECIDE -D MIXPANEL_RANDOM_DISTINCT_ID'
>               end
>           end
>       end
>   end
> ```
> 
> And don't forget... `Command-Option-Shift-K` to make your problems go away!

Thank you a lot. It works for me.

**Comment by zihejia** [comment]
Hi @IsaiahJTurner @IsaiahJTurner , thanks for bringing it up. We have released [v2.7.7 ](https://github.com/mixpanel/mixpanel-swift/releases/tag/v2.7.7,) , in preparation of the coming iOS 14 release, we no longer use IDFA and generate random distinct id by default.
Updated docs: 
[iOS - Swift developer doc](https://developer.mixpanel.com/docs/swift), 
[Distinct ID Creation](https://help.mixpanel.com/hc/en-us/articles/115004509426-Distinct-ID-Creation-JavaScript-iOS-Android-)

I'm closing this one now, feel free to raise any questions or reach out to Mixpanel support.
